test/Mono.Linker.Tests/Tests/PreserveActionComparisonTests.cs

```diff
@@ -20,7 +20,11 @@ public class PreserveActionComparisonTests
 		[TestCase (TypePreserve.Nothing, TypePreserve.All, TypePreserve.All)]
 		[TestCase (TypePreserve.Nothing, TypePreserve.Methods, TypePreserve.Methods)]
 		[TestCase (TypePreserve.Nothing, TypePreserve.Fields, TypePreserve.Fields)]
-		[TestCase (TypePreserve.Nothing, TypePreserve.Nothing, TypePreserve.Nothing)]
+		[TestCase (TypePreserve.All, TypePreserve.All | TypePreserve.AccessibilityVisible, TypePreserve.All)]
+		[TestCase (TypePreserve.AccessibilityVisibleOrInternal, TypePreserve.All, TypePreserve.All)]
+		[TestCase (TypePreserve.All | TypePreserve.AccessibilityVisible, TypePreserve.All | TypePreserve.AccessibilityVisible, TypePreserve.All | TypePreserve.AccessibilityVisible)]
+		[TestCase (TypePreserve.Nothing, TypePreserve.All | TypePreserve.AccessibilityVisible, TypePreserve.All | TypePreserve.AccessibilityVisible)]
+		[TestCase (TypePreserve.All | TypePreserve.AccessibilityVisibleOrInternal, TypePreserve.Nothing, TypePreserve.All | TypePreserve.AccessibilityVisibleOrInternal)]
 		public void VerifyBehaviorOfChoosePreserveActionWhichPreservesTheMost (TypePreserve left, TypePreserve right, TypePreserve expected)
```

Comment:
Does this need to handle a case like (TypePreserve.All | TypePreserve.AccessibilityVisible, TypePreserve.Fields)?

Reply:
No, this is not needed/possible scenario